FABulous/FABulous_CLI/FABulous_CLI.py

```diff
@@ -326,6 +327,34 @@ def do_install_oss_cad_suite(self, args: argparse.Namespace) -> None:
 
         install_oss_cad_suite(dest_dir, args.update_existing)
 
+    install_FABulator_parser = Cmd2ArgumentParser()
+    install_FABulator_parser.add_argument(
+        "destination_folder",
+        type=Path,
+        help="Destination folder for the installation",
+        default="",
+        completer=Cmd.path_complete,
+        nargs=argparse.OPTIONAL,
+    )
+
+    @with_category(CMD_SETUP)
+    @allow_blank
```

Comment:
You can keep this for now, but we can stop using it if we make sure all the calls are done via the `onecmd_plus_hooks`, which can avoid the blank issue. If I recall correctly, the upstream `cmd2` has also fixed this problem.